plugins/uribl

```diff
@@ -265,8 +265,9 @@ sub lookup_start {
     my %pending;
     my @qp_continuations;
     
-    # Prepare the TLD list in __DATA__ section for use in RegEx
-    (my $tlds4regex = join('|',  grep(/\S/, <DATA>))) =~ s/\R//g;
+    # Fetch the TLD list and prepare it for use in RegEx
+    my @tlds = $self->qp->config('uribl_tlds');
+    (my $tlds4regex = join('|',  grep(/\S/, <@tlds>))) =~ s/\R//g;
 
```

Comment:
Since we don't reuse that array, might it be better (re: more memory efficient) to inline it and store only `$tlds4regex`?